src/Smalot/PdfParser/Page.php

```diff
@@ -799,6 +823,10 @@ public function getDataTm(array $dataCommands = null): array
                     $extractedData[] = [$Tm, $data[2]]; //Verify
                     break;
 
+                case 'Tf':
+                    list($fontId, $fontSize) = explode(' ', $command['c'], 2);
```

Comment:
Is a further check here required? Or can we assume that `$command['c']` always has first two elements set?